$MirOS: ports/print/cups/patches/patch-monitor_Makefile,v 1.1 2006/06/21 19:25:35 bsiegert Exp $
--- monitor/Makefile.orig	Wed Jul 11 23:46:19 2007
+++ monitor/Makefile	Tue Oct 21 22:56:23 2008
@@ -80,7 +80,7 @@ uninstall:
 
 bcp:	bcp.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ bcp.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ bcp.o $(LIBS)
 
 
 #
@@ -89,7 +89,7 @@ bcp:	bcp.o ../cups/$(LIBCUPS)
 
 tbcp:	tbcp.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ tbcp.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ tbcp.o $(LIBS)
 
 
 #
